README-Bazel.md

```diff
@@ -0,0 +1,27 @@
+# WPILib Bazel Support
+
+WPILib is normally built with Gradle, but [Bazel](https://www.bazel.build/) can also be used to increase development speed due to the superior caching ability and the ability to use remote caching and remote execution (on select platforms)
+
+
+## Prerequisites
+- Install [Bazelisk](https://github.com/bazelbuild/bazelisk/releases) and add it to your path. Bazelisk is a wrapper that will download the correct version of bazel specified in the repository. Note: You can alias/rename the binary to `bazel` if you want to keep the familiar `bazel build` vs `bazelisk build` syntax.
+- Java8+. The build will hermetically use java11, but bazel requires some version of the JRE be installed to run.
```

Comment:
I thought Bazelisk usually downloads a build of Bazel that ships with its own JVM.

Reply:
I'll admit I copied this from my ~3 year old pull request to add bazel the first time. I missed some of removals that I either got wrong the first time or are no longer applicable.